strongbox-storage/strongbox-storage-api/src/test/java/org/carlspring/strongbox/testing/repository/RawRepositoryManagementTest.java

```diff
@@ -0,0 +1,59 @@
+package org.carlspring.strongbox.testing.repository;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.carlspring.strongbox.StorageApiTestConfig;
+import org.carlspring.strongbox.storage.repository.Repository;
+import org.carlspring.strongbox.testing.NullLayoutProvider;
+import org.carlspring.strongbox.testing.storage.repository.RepositoryManagementTestExecutionListener;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.api.parallel.Execution;
+import org.junit.jupiter.api.parallel.ExecutionMode;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.ContextConfiguration;
+
+/**
+ * @author ankit.tomar
+ */
+@SpringBootTest
+@ActiveProfiles(profiles = "test")
+@ContextConfiguration(classes = StorageApiTestConfig.class)
+@Execution(ExecutionMode.CONCURRENT)
+public class RawRepositoryManagementTest
+{
+
+    private static final String RAW_STORAGE = "raw-storage-to-be-injected";
+
+    private static final String RAW_REPOSITORY = "rwa-repository-to-be-injected";
+
+    @ExtendWith({ RepositoryManagementTestExecutionListener.class })
+    @Test
+    public void testValidRepositoryShouldBeInjected(@NullRepository(repositoryId = RAW_REPOSITORY,
+                                                                    storageId = RAW_STORAGE)
```

Comment:
Cosmetic: Could you please swap the places of the `storageId` and `repositoryId` parameters? (Logically, first comes the storage and then the repositories are inside of it). Thanks! :)